coregames/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: core-games, core-content, core-communitycontent, core-community-content, core-cc, core-scripts, core-editor, core-development, core-engine, core-gameengine, core-game-engine, core-platform, manticore, manticore-games
```

Comment:
These aliases are for topics that either aren't used yet ([core-games](https://github.com/topics/core-games), [core-content](https://github.com/topics/core-content)), or are too broad ([core-platform](https://github.com/topics/core-platform), [manticore](https://github.com/topics/manticore)) - that is, they seem to be applied to many repositories that don't actually have to do with the Core game engine.

```suggestion
```

Let's delete these before we merge this. Instead of adding aliases to topics that we think repositories might use in the future to affiliate themselves with a curated topic area, we tend to add them when we see repositories using them already instead (like `react` and `reactjs`).